### PR TITLE
Fix constant_value in imputer

### PR DESCRIPTION
**Pull request by julia-shenshina**
## Before submitting (must do checklist)

- [x] Did you read the [contribution guide](https://github.com/tinkoff-ai/etna/blob/master/CONTRIBUTING.md)?
- [x] Did you update the docs? We use Numpy format for all the methods and classes.
- [x] Did you write any new necessary tests?
- [x] Did you update the [CHANGELOG](https://github.com/tinkoff-ai/etna/blob/master/CHANGELOG.md)?

### CODE REVIEW

File: etna/transforms/missing_values/imputation.py [hunk 1]
```diff
@@ -86,7 +88,7 @@ def __init__(
         self.window = window
         self.seasonality = seasonality
         self.default_value = default_value
-        self.fill_value: Optional[int] = None
+        self.fill_value: Optional[float] = None
```
Comment (alex-hse-repository) [inline etna/transforms/missing_values/imputation.py:91]: Let's place the constant_value initialization before this line

File: tests/test_transforms/test_missing_values/test_impute_transform.py [hunk 2]
```diff
@@ -78,25 +78,37 @@ def test_all_missing_impute_fail_two_segments(df_all_missing_two_segments: pd.Da
         _ = imputer.fit_transform(df_all_missing_two_segments)
 
 
-def test_one_missing_value_zero(df_with_missing_value_x_index: pd.DataFrame):
-    """Check that imputer with constant-strategy with zero value correctly in case of one missing value in data."""
+@pytest.mark.parametrize("constant_value", (0, 42))
+def test_one_missing_value_zero(df_with_missing_value_x_index: pd.DataFrame, constant_value: float):
```
Comment (alex-hse-repository) [inline tests/test_transforms/test_missing_values/test_impute_transform.py:82]: test_one_missing_value_constant

**Comment by alex-hse-repository** [comment]
Also up the version in the depreciation warning to 2.0.0

**Review by alex-hse-repository — CHANGES_REQUESTED** [review]

File: etna/transforms/missing_values/imputation.py (hunk 1 above)
Comment (julia-shenshina) [reply etna/transforms/missing_values/imputation.py:91]: https://github.com/tinkoff-ai/etna/pull/819/files#diff-345da9722fb55e4c74ad4e4c73fdc9eca093844b5705985c38bd4b5272e4c774R87
?

**Review by alex-hse-repository — APPROVED** [review]

**Pull request merged**